docs/notif/ntfy.md

```diff
@@ -0,0 +1,52 @@
+# Ntfy notifications
+
+Notifications can be sent using a [ntfy](https://ntfy.sh/) instance.
+
+## Configuration
+
+!!! example "File"
+    ```yaml
+    notif:
+      ntfy:
+        endpoint: https://ntfy.sh
+        topic: diun-acce65a0-b777-46f9-9a11-58c67d1579c4
+        priority: default
+        emoji: whale
+        timeout: 10s
+        templateTitle: "{{ .Entry.Image }} released"
+        templateBody: |
+          Docker tag {{ .Entry.Image }} which you subscribed to through {{ .Entry.Provider }} provider has been released.
+    ```
+
+| Name                | Default                             | Description                                                                |
+| ------------------- | ----------------------------------- | -------------------------------------------------------------------------- |
+| `endpoint`[^1]      |                                     | Ntfy base URL                                                              |
+| `topic`             |                                     | Ntfy topic                                                                 |
+| `priority`          | `default`                           | The priority of the message                                                |
+| `timeout`           | `10s`                               | Timeout specifies a time limit for the request to be made                  |
+| `templateTitle`[^1] | See [below](#default-templatetitle) | [Notification template](../faq.md#notification-template) for message title |
+| `templateBody`[^1]  | See [below](#default-templatebody)  | [Notification template](../faq.md#notification-template) for message body  |
+| `emoji`             | `package`                           | Emoji to go in your notiication                                            |
+
+!!! abstract "Environment variables"
+    * `DIUN_NOTIF_NTFY_ENDPOINT`
+    * `DIUN_NOTIF_NTFY_TOPIC`
+    * `DIUN_NOTIF_NTFY_PRIORITY`
+    * `DIUN_NOTIF_NTFY_TIMEOUT`
+    * `DIUN_NOTIF_NTFY_TEMPLATETITLE`
+    * `DIUN_NOTIF_NTFY_TEMPLATEBODY`
+    * `DIUN_NOTIF_NTFY_EMOJI`
```

Comment:
For consistency, `DIUN_NOTIF_NTFY_EMOJI` should be placed before `DIUN_NOTIF_NTFY_TIMEOUT`. And also renamed `DIUN_NOTIF_NTFY_TAGS`.